test/system/160-volumes.bats

```diff
@@ -556,7 +556,11 @@ EOF
 
 @test "podman volume rm --force bogus" {
     run_podman 1 volume rm bogus
-    is "$output" "Error: no volume with name \"bogus\" found: no such volume" "Should print error"
+    expectedoutput="Error: no such volume"
+    if [[ "$CI_DESIRED_DATABASE" = "boltdb" ]]; then
```

Comment:
Saw this env var used in some other `test/system/*.bats` files, wasn't sure if it was OK to use it here?